cuegui/cuegui/config/cue_config.yaml

```diff
@@ -0,0 +1,3 @@
+# Internal specs/variables that could also be overriden using environment variables
+
+sentry_dsn_path: ''
```

Comment:
What happens if this doesn't get set? 

The comment on this config entry should specify what happens and provide a link to sentry's docs

Reply:
If no DSN path is set it will output this warning in the shell (log levels are set at warning level):
`Main.py:No Sentry DSN path found. Skipping Sentry setup`

I can change the log level to info so it will not output to the shell?